### ISSUE TITLE
HDR Streams are Washed Out

### ISSUE THREAD

**Issue by ChaoDrake** [Stremio/stremio-bugs#414]
**Describe the bug**
Any 4K HDR streams are washed out (very dark and muted colors) compared to the official streams (i.e. Netflix, Disney+, etc.)

**To Reproduce**
1. Go to any 4K HDR Stream (i.e. House of Dragons S01E05)
2. Click on play.
3. Scroll down to stream.
4. See that the colors are washed out, which are especially apparent when playing a non HDR video in comparison.

**Expected behavior**
HDR video shouldn't look so washed out compared to a regular video stream, this problem doesn't arise in the official streaming apps.

**Screenshots**
No screenshots.

**Smartphone (please complete the following information):**
 - Device: Chromecast with Google TV
 - OS: Android OS
 - Browser No Browser
 - Version 1.5.6 (Latest Android TV version)

**Additional context**
I've also tried this on my Google Pixel 6 Pro and it also seems to have the same issue, so I'm not sure what's going on here. FYI I am using Torrentio with Real Debrid as the addon to find the streams.

**Comment by TRtomasz** [comment]
Most likely you messed up with your hardware accelerated/tunneled playback settings on your TV. You should try different config

**Comment by ChaoDrake** [comment]
If you're talking about the stremio app settings. I've tried all the different configs of hardware accel on/off and tunneled playback on/off but nothing changed. When enabling tunneled playback, all videos are just a black screen on my Chromecast, and when turning hardware accel off, the videos stutter really badly.

If talking about TV or Chromecast settings, I'm not sure there is hardware acceleration/tunneled playback settings, not one that I could find anyway. In Chromecast settings, I've tried it always on Dolby Vision, and always on HDR, but again, none seem to affect that washed out look.

**Comment by TRtomasz** [comment]
does your tv support hdr?

**Comment by ChaoDrake** [comment]
Yes, it is a TCL 55R617, and supports HDR10 and Dolby Vision (the logo pops up on the top right corner when I'm on the Chromecast). I've tried the Chromecast display settings as you've said, and still the same issue.

**Comment by nipmarsh** [comment]
I noticed the same issue too, I use a Vava Chroma 4K HDR10 laser projector, compared to SDR streams the imaged is really washed

**Comment by roisec** [comment]
I have the same problem too.
Xiaomi mi box
